### PR TITLE
Add logical assignment operators

### PR DESCRIPTION
**Pull request by Elchi3**
Examples for 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND_assignment
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

Quite unsure how AND assignment is useful (at all).

### CODE REVIEW

File: live-examples/js-examples/expressions/expressions-logical-nullish-assignment.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let a = { duration: 50 };
```
Comment (wbamberg) [inline live-examples/js-examples/expressions/expressions-logical-nullish-assignment.js:1]:
```suggestion
const a = { duration: 50 };
```

File: live-examples/js-examples/expressions/expressions-logical-or-assignment.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+let a = { duration: 50, title: '' };
```
Comment (wbamberg) [inline live-examples/js-examples/expressions/expressions-logical-or-assignment.js:1]:
```suggestion
const a = { duration: 50, title: '' };
```

File: live-examples/js-examples/expressions/expressions-logical-and-assignment.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+let a = 1;
+
+a &&= 1;
+console.log(a);
+// expected output: 1
+
+a &&= 0;
+console.log(a);
+// expected output: 0
```
Comment (wbamberg) [inline live-examples/js-examples/expressions/expressions-logical-and-assignment.js:9]: As I read the docs, this operator behaves differently depending on the value of the left-hand side. But this example uses just one lhs value and two different rhs values. As such, it doesn't show the difference between `&&=` and plain `=`.

What if you have two different lhs values and one rhs value, like:

```js
let a = 1;
let b = 0;

a &&= 2;
console.log(a);
// expected output: 2

b &&= 2;
console.log(b);
// expected output: 0
```

Comment (Elchi3) [reply live-examples/js-examples/expressions/expressions-logical-and-assignment.js:10]: Yeah, that makes sense. I have this in the static example on the page, too.

**Comment by Elchi3** [comment]
Updated as well, thanks for the review, Will! :+1:

**Pull request merged**